### ISSUE TITLE
OpenAPI Validation Issues

### ISSUE THREAD

**Issue by mneagul** [opengeospatial/ogcapi-processes#256]
While trying to generate stubs or validate the current schema using the various tools available experience errors related to missing references. For example:

```
docker run --volume "$PWD":/data jamescooke/openapi-validator  openapi.yaml                                                                                    (docker-desktop/default)

[Error] There is a problem with the Swagger.
Error downloading https://raw.githubusercontent.com/opengeospatial/ogcapi-processes/master/core/openapi/schemas/result.yaml
HTTP ERROR 404
```

This is in the route `/processes/{processId}/execution`, more exactly in the callbacks section.

**Comment by gfenoy** [comment]
May I ask from where the `openapi.yaml` file you are referring to is coming from? For further investigations, it would definitely help to have access to the `openapi.yaml file you are using, or at least the faulty part.

In case it is of any help, an example is provided in [core/examples/openapi/ogcapi-processes-1-example.yaml](https://github.com/opengeospatial/ogcapi-processes/blob/master/core/examples/openapi/ogcapi-processes-1-example.yaml). 

I suppose that the issue here is only related to an old API use for your testing. In this API, there is a reference to a file from the schema which is not available anymore. Indeed, there is no result.yaml but a [results.yaml](https://raw.githubusercontent.com/opengeospatial/ogcapi-processes/master/core/openapi/schemas/results.yaml) file. So, it may only be a typo in the API definition. The results.yaml file should be used for defining the jobSuccess callback requestBody content schema. You may take a look at this tentative [API definition](http://tb17.geolabs.fr:8100/ogc-api/api.html) and see the results.yaml file used in the callbacks section from the /api path.

**Comment by gfenoy** [comment]
In case you are only referring to the "OGC API - Processes on Swaggerhub" link from the README.md file, please consider it as outdated and use this one [here](https://app.swaggerhub.com/apis/ZOO-Project/the-zoo_project_wps_ogc_api_processing_development_server/1.0.0-mneagul).

**Comment by mneagul** [comment]
Hi Gérald,

The `openapi.yaml` is the one from the root of this repository: https://github.com/opengeospatial/ogcapi-processes/blob/master/openapi.yaml

Marian

**Comment by bpross-52n** [comment]
I fixed the wrong URL and I will check the yaml file with the validator that you use.

**Comment by bpross-52n** [comment]
This should be fixed by #277
